backend/internal/cmds/mgmt/serve/connect/cmd.go

```diff
@@ -165,7 +165,11 @@ func serve(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		apikeyClient := auth_apikey.New(db.Q, db.Db)
+		apikeyClient := auth_apikey.New(db.Q, db.Db, getAllowedWorkerApiKeys(getIsNeosyncCloud()), []string{
+			mgmtv1alpha1connect.JobServiceGetJobProcedure,
+			mgmtv1alpha1connect.ConnectionServiceGetConnectionProcedure,
+			mgmtv1alpha1connect.TransformersServiceGetUserDefinedTransformerByIdProcedure,
```

Comment:
when providing a worker api key, these are the only methods that they are valid for.